aci/resource_aci_rtctrlattrp.go

```diff
@@ -1439,8 +1547,6 @@ func resourceAciActionRuleProfileRead(ctx context.Context, d *schema.ResourceDat
 			return nil
 		}
 		log.Printf("[DEBUG] %s: rtctrlSetComm - Read finished successfully", setCommDn)
-	} else {
-		d.Set("set_communities", nil)
```

Comment:
Why removing this? How do you deal with set_communities being unset externally and not detected by Terraform?